honeybadger.go

```diff
@@ -0,0 +1,83 @@
+package cleisthenes
+
+type HoneyBadger struct {
+
+	// TODO : HoneyBadger must have ACS
+
+	// TODO : HoneyBadger must have consensused batch
+
+	// que have transaction FIFO queue.
+	que queue
+
+	// current epoch
+	epoch int
+}
+
+// Add transaction to FIFO queue
+func (hb *HoneyBadger) AddTransaction() {
+	panic("implement me w/ test case :-)")
+}
+
+// Send batch to ACS
+func (hb *HoneyBadger) SendBatch() {
```

Comment:
I think this function will not needed. sending batch will be called inside inifinite-loop (asynchronously consensusing)

Reply:
I think it is ok to test or use go routine. how about define as private function?

Reply:
@AgwaB We agree what you said :-)